plugins/Diagnostics/plugin.json

```diff
@@ -1,3 +1,6 @@
 {
-  "description": "Performs diagnostics to check that Piwik is installed and runs correctly."
+    "description": "Performs diagnostics to check that Piwik is installed and runs correctly.",
+    "require": {
+        "piwik": ">=2.16.0-b2"
```

Comment:
That wasn't supposed to be here. It's kinda a problem with the generator command. It should not set it for core plugins.